docker/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.1'
+
+services:
+  identities_db:
+    image: postgres:11-alpine
+    volumes:
+      - $HOME/docker/volumes/postgres:/var/lib/postgresql/data
```

Comment:
can we use `/opt` for this purpose?

Reply:
oh, we cannot. I would like to use it on Windows. Let's make it persistent volume.

Reply:
Check out this link https://docs.docker.com/compose/compose-file/#volumes